catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/QueryOperations.java

```diff
@@ -473,9 +487,12 @@ private QueryResponse populateQueryResponsePolicyMap(QueryResponse queryResponse
   }
 
   private QueryRequest processPreQueryPlugins(QueryRequest queryReq) throws FederationException {
+    String traceId = ThreadContextProperties.getTraceId();
     for (PreQueryPlugin service : frameworkProperties.getPreQuery()) {
       try {
+        LOGGER.trace("Before PreQueryPlugin: trace-id {}, plugin {}", traceId, service);
```

Comment:
Does logging the service object provide the details we care about?  I assume this will display the name of the implementing class of the PreQueryPlugin interface which is probably sufficient.